vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -92,7 +91,6 @@ switch ($windowsSKU) {
             "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v1.23.2", # for k8s 1.23.x
             "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v1.23.5", # for k8s 1.23.x
             "mcr.microsoft.com/azuremonitor/containerinsights/ciprod:win-ciprod01312022",
-            "mcr.microsoft.com/azuremonitor/containerinsights/ciprod:win-ciprod03112022",
             "mcr.microsoft.com/azuremonitor/containerinsights/ciprod:win-ciprod03172022")
     }
     "2022-containerd" {
```

Comment:
dont we need to add the omsagent images under 2012-containerd ?

Reply:
Yes. 2022-containerd does not need omsagent images

Reply:
> Yes. 2022-containerd does not need omsagent images

Ok thanks.

Reply:
@ganga1980 omsagent-win does not support WS2022 for now so that is why we remove them.

Reply:
> @ganga1980 omsagent-win does not support WS2022 for now so that is why we remove them.

> make sense. thanks Abel.